src/sys.rs

```diff
@@ -31,16 +31,16 @@ pub enum Mode {
     /// Level-triggering
     ///
     /// This FD will report events on every poll as long as the requested interests
-    /// are available. If the same FD is inserted in multiple event loops, all of
-    /// them are notified of readiness.
+    /// are available.
     Level,
 
     /// Edge-triggering
     ///
     /// This FD will report events only when it *gains* one of the requested interests.
-    /// it must thus be fully processed before it'll generate events again. If the same
-    /// FD is inserted on multiple event loops, it may be that not all of them are notified
-    /// of readiness, and not necessarily always the same(s) (at least one is notified).
+    /// it must thus be fully processed before it'll generate events again.
+    ///
+    /// This mode is not supported on certain platforms, and an error will be returned
+    /// if it is used.
```

Comment:
In that case, we should either document which platform support it, or provide a way to check if it is supported at runtime.